apps/accounts/migrations/0003_affiliation_remove.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
@HargovindArora Can you please rename the migration as `remove_affiliation_model.py`

Reply:
@RishabhJain2018 Sure :smile: